herd/variant.ml

```diff
@@ -132,6 +135,8 @@ let parse s = match Misc.lowercase s with
 | "constrainedunpredictable"|"cu" -> Some ConstrainedUnpredictable
 | "exp" -> Some Exp
 | "self" -> Some Self
+| "dic" -> Some DIC
+| "idc" -> Some IDC
```

Comment:
I agree that it would be good to prevent specifying DIC/IDC as variants.

My quick impression is that your suggestion would be very neat, but it would also prevent the cat interpreter from parsing DIC/IDC in cat files. I will look into how to achieve one without the other and get back to you.